lib/cc/analyzer/engine.rb

```diff
@@ -34,8 +34,12 @@ def run(stdout_io, container_listener)
         )
 
         container.on_output("\0") do |output|
-          unless output_filter.filter?(output)
-            stdout_io.write(output) || container.stop
+          next if output.blank?
+
+          issue = Issue.new(output)
+
+          unless output_filter.filter?(issue)
```

Comment:
Doh, I guess this should be `issue_filter` now. I'm fine punting on the rename for now, it'd muddy the diff quite a bit I think.